docs/core_docs/docs/integrations/document_loaders/web_loaders/google_cloud_pg.mdx

```diff
@@ -0,0 +1,106 @@
+---
+hide_table_of_contents: true
+---
+
+import loadExample from "@examples/document_loaders/postgres_load";
+import CodeBlock from "@theme/CodeBlock";
+
+# PostgresLoader
+
+The `PostgresLoader` class provides functionality to load documents from a PostgreSQL database. It supports two main methods:
+
+1. **`load()`**: Retrieves all documents at once, ideal for small to moderate datasets.
+2. **`lazyLoad()`**: Fetches documents one by one, which is more memory-efficient for large datasets.
+
+## Prerequisites
+
+Before using the `PostgresLoader`, ensure that you have the following:
+
+1. A **PostgreSQL database** with the necessary tables and data.
+2. A **PostgresEngine** instance configured to connect to your database.
+3. The required environment variables for database authentication (e.g., `DB_USER`, `PASSWORD`).
+
+## Usage
+
+### Example: Loading Documents from a Table
+The following example demonstrates how to use the `PostgresLoader` to load documents from a PostgreSQL table.
+
+<CodeBlock language="typescript">{loadExample}</CodeBlock>
```

Comment:
loadExample links to "import loadExample from "@examples/document_loaders/postgres_load";". Is this the correct link to code?